### PR TITLE
isEdit don't use on the server. Work with the server has been normal

### PR DESCRIPTION
**Pull request by MkpDolganovKirill**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,32 +1,70 @@
-let allTasks = JSON.parse(localStorage.getItem('tasks')) || [];
+let allTasks = [];
 let valueInput = '';
 let input = null;
 
-window.onload = () => {
+window.onload = async () => {
 	input = document.getElementById('input-task');
```
Comment (DoctorFreed) [inline script.js:6]: indent 2 spaces

File: script.js [hunk 2]
```diff
@@ -1,32 +1,70 @@
...
+			allTasks.forEach(element => {
+				element.isEdit = false;
+			});
+		};
 	} else {
 		alert("You can't add empty task!");
 	};
-	saveToLocalStorage();
+
+
```
Comment (DoctorFreed) [inline script.js:53]: remove empty strings

File: script.js [hunk 3]
```diff
@@ -108,43 +144,72 @@ render = () => {
...
+const deleteElement = async (index) => {
+	const resp = await fetch(`http://localhost:8000/deleteTask?id=${allTasks[index].id}`, {
+		method: 'DELETE'
+	});
+	let result = await resp.json();
+	allTasks = result.data;
 	render();
 };
 
 const sortedTasks = () => {
```
Comment (DoctorFreed) [inline script.js:202]: change sortedTasks method, he can be much less

File: script.js [hunk 4]
```diff
@@ -1,32 +1,70 @@
-let allTasks = JSON.parse(localStorage.getItem('tasks')) || [];
+let allTasks = [];
 let valueInput = '';
 let input = null;
 
-window.onload = () => {
+window.onload = async () => {
 	input = document.getElementById('input-task');
 	input.addEventListener('change', updateValue);
+	const resp = await fetch('http://localhost:8000/allTasks', {
```
Comment (omanei) [inline script.js]: indent 2 spaces

File: script.js (hunk 2 above)
Comment (omanei) [inline script.js]: remove empty strings

File: script.js (hunk 3 above)
Comment (omanei) [inline script.js:199]: change sortedTasks method, he can be much less

**Pull request merged**